diff --git a/node_modules/preact/src/internal.d.ts b/node_modules/preact/src/internal.d.ts
index 60e8d63..96fc2db 100644
--- a/node_modules/preact/src/internal.d.ts
+++ b/node_modules/preact/src/internal.d.ts
@@ -1,6 +1,5 @@
 import * as preact from './index';
 
-declare global {
 	export enum HookType {
 		useState = 1,
 		useReducer = 2,
@@ -191,4 +190,3 @@ declare global {
 		_id: string;
 		_defaultValue: any;
 	}
-}
